test/js/samples/component-store-access-invalidate/expected.js

```diff
@@ -43,7 +43,7 @@ function instance($$self, $$props, $$invalidate) {
 	let $foo;
 	const foo = writable(0);
 	component_subscribe($$self, foo, value => $$invalidate(0, $foo = value));
-	return [$foo];
+	return [$foo, foo];
```

Comment:
I don't think it'll cause any incorrect behavior, but it would be nice if this weren't added. Presumably the bitmask changes co-opted `referenced` to mean 'needed in the context array', which is also what caused this bug in the first place when that started being false for stores that are only accessed via autosubscription in the template. That may not be worth it to fix at this point though.

Reply:
Here's something interesting though... 
If you add another normal variable that can't be hoisted, it gets put in the context list anyway.
This occurs in both current master as well as my branch.
https://svelte.dev/repl/e61b08c7da904ba7af9d005ebac0d2d2?version=3.16.3

So, yeah, it probably isn't a big enough issue to worry about.